packages/@sanity/form-builder/src/inputs/BlockEditor-slate/FormBuilderBlock.js

```diff
@@ -299,17 +303,46 @@ export default class FormBuilderBlock extends React.Component {
       )
     }
 
+    if (editModalLayout === 'popover') {
+      return (
+        <div className={styles.editBlockContainerPopOver}>
+          <EditItemPopOver
+            isOpen
+            title={this.props.node.title}
+            onClose={this.handleClose}
+          >
+            {input}
+          </EditItemPopOver>
+        </div>
+      )
+    }
+
     // default
     return (
-      <div className={styles.editBlockContainerPopOver}>
-        <EditItemPopOver
-          isOpen
-          title={this.props.node.title}
-          onClose={this.handleClose}
-        >
-          {input}
-        </EditItemPopOver>
-      </div>
+      <DefaultDialog
+        isOpen
+        title={this.props.node.title}
+        onClose={this.handleClose}
+        showCloseButton={false}
+        onAction={this.handleDialogAction}
+        actions={[
+          {
+            index: '1',
+            name: 'close',
+            title: 'Close'
+          },
+          // {
```

Comment:
I suggest you hoist these too, and maybe add a todo about why the delete action is commented out?

Reply:
👍